benchmarks/dynamo/torchbench.py

```diff
@@ -77,6 +77,7 @@ def setup_torchbench_cwd():
     # TIMEOUT, https://github.com/pytorch/pytorch/issues/98467
     "tacotron2",
 }
+SKIP.update(DETECTRON2_MODELS)
```

Comment:
https://github.com/pytorch/pytorch/actions/runs/5480430769/jobs/9984339567

Reply:
https://github.com/facebookresearch/detectron2/commit/ff53992b1985b63bd3262b5a36167098e3dada02 should resolve this.

Reply:
Please do follow up on this. Thanks!

Reply:
Or after #104760